spec/features/exercise_flow_spec.rb

```diff
@@ -9,7 +9,9 @@
   let(:problem_1) { build(:problem, name: 'Succ1', description: 'Description of Succ1', layout: :input_right, hint: 'lala') }
   let(:problem_2) { build(:problem, name: 'Succ2', description: 'Description of Succ2', layout: :input_right, editor: :hidden, language: gobstones) }
   let(:problem_3) { build(:problem, name: 'Succ3', description: 'Description of Succ3', layout: :input_right, editor: :upload, hint: 'lele') }
-  let(:problem_4) { build(:problem, name: 'Succ4', description: 'Description of Succ4', layout: :input_bottom, extra: 'x = 2') }
+  let(:with_settings_and_extra) do
```

Comment:
Those tests need a refactor. They do a lot of things, are heavy, and their object names are not intention revealing